package/InternalEAWGen.roc

```diff
@@ -0,0 +1,140 @@
+## The purpose of this file is to generate the InternalEAW.roc file.
+##
+## This file will read the test data from `data/EastAsianWidth-15.1.0.txt`
+## parse it and then generate function to test the East Asian Width property of a code point.
+app [main] {
+    pf: platform "https://github.com/roc-lang/basic-cli/releases/download/0.11.0/SY4WWMhWQ9NvQgvIthcv15AUeA7rAIJHAHgiaSHGhdY.tar.br",
+}
+
+import pf.File
+import pf.Task exposing [Task]
+import pf.Arg
+import "data/EastAsianWidth-15.1.0.txt" as file : Str
+import Helpers
+
+EawRange : (Str, Str, Str)
+
+main =
+    when Arg.list! |> List.get 1 is
+        Err _ -> Task.err (InvalidArguments "USAGE: roc run InternalEAWGen.roc -- path/to/package/")
+        Ok arg -> File.writeUtf8 "$(Helpers.removeTrailingSlash arg)/InternalEAW.roc" template
```

Comment:
Upgraded to basic-cli 0.11.0, we can simplify this a lot. The other Gen scripts were written a while ago and a lot has changed since then. For example we can use the `Arg.list!` inline here which I think is nice.